bigtable/tests/unit/test_row.py

```diff
@@ -190,31 +180,23 @@ def __init__(self, *args, **kwargs):
                 self._args = []
                 self._kwargs = []
 
-            # Replace the called method with one that logs arguments.
-            def _delete_cells(self, *args, **kwargs):
-                self._args.append(args)
-                self._kwargs.append(kwargs)
-
```

Comment:
Without the `_delete_cells` class, the entire `MockRow` bit is pointless, and should be deleted:  the test should just use an instance of the real class.

Reply:
@aneepct and @zakons, should we remove MockRow?